### ISSUE TITLE
what is lfw.txt in test.py and from where can i access that for testing the model??

### ISSUE THREAD

**Issue by mehtajinesh** [guoyilin/FaceDetection_CNN#12]
After training the model, I get the loss to be very high(can you tell me how to decrease it).
Following is the output after 10000 iter of training that i got:

I0331 04:21:06.584506  2709 sgd_solver.cpp:273] Snapshotting solver state to binary proto file /media/jinesh/ACB6A202B6A1CCE0/Academics/8thSemester/CNN/train_iter_100000.solverstate
I0331 04:21:08.205349  2709 solver.cpp:311] Iteration 100000, loss = 0.00238461
I0331 04:21:08.205370  2709 solver.cpp:331] Iteration 100000, Testing net (#0)
I0331 04:22:55.422292  2709 solver.cpp:398]     Test net output #0: accuracy = 0.993758
I0331 04:22:55.422312  2709 solver.cpp:316] Optimization Done.
I0331 04:22:55.464334  2709 caffe.cpp:259] Optimization Done.

Next for testing I opened test.py but it says:
if __name__ == "__main__":
    #convert_full_conv()
    face_detection("lfw.txt")
so where is lfw.txt in the repository?

**Comment by mehtajinesh** [comment]
ok figured it out.
